src/main/java/org/gridsuite/explore/server/services/FilterService.java

```diff
@@ -80,15 +82,16 @@ public void setFilterServerBaseUri(String filterServerBaseUri) {
                 .log(ROOT_CATEGORY_REACTOR, Level.FINE);
     }
 
-    public Mono<Void> deleteFilter(UUID id) {
+    public Mono<Void> delete(UUID id, String userId) {
         String path = UriComponentsBuilder.fromPath(DELIMITER + FILTER_SERVER_API_VERSION + "/filters/{id}")
                 .buildAndExpand(id)
                 .toUriString();
 
         return webClient.delete()
                 .uri(filterServerBaseUri + path)
+                .header(HEADER_USER_ID, userId)
                 .retrieve()
-                .onStatus(httpStatus -> httpStatus != HttpStatus.OK, r -> Mono.empty())
+                .onStatus(httpStatus -> httpStatus != HttpStatus.OK, r -> Mono.error(() -> new ExploreException(ExploreException.Type.NOT_ALLOWED)))
```

Comment:
Why is it different than the not found above ?

Reply:
hramonized